src/V3DfgBreakCycles.cpp

```diff
@@ -580,22 +580,29 @@ class TraceDriver final : public DfgVisitor {
 
 class IndependentBits final : public DfgVisitor {
     // STATE
-    DfgVarPacked* const m_varp;  // The variable we are computing dependencies for
-    std::deque<DfgVertex*> m_workList;  // Work list for traversal
+    const uint32_t m_component;  // The component the start vertex is part of
     // Vertex to current bit mask map. The mask is set for the bits that **depend** on 'm_varp'.
     std::unordered_map<const DfgVertex*, V3Number> m_vtxp2Mask;
 
     std::ofstream m_lineCoverageFile;  // Line coverage file, just for testing
 
     // METHODS
+    static void setRangeZero(V3Number& num, uint32_t msb, uint32_t lsb) {
+        for (uint32_t i = lsb; i <= msb; ++i) num.setBit(i, '0');
+    }
```

Comment:
Please make a V3Number::opSetRangeZero, to match opSelInto probably better with lsb & width instead of lsb & msb.

Reply:
Done, but I'm not sure it quite fits there. Resolve if you are happy.